R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
+#' @param type The default option is \code{type = "facet"}. If one model only is 
+#'   specified, it will print one forest plot applied to point estimates and 
+#'   confidence intervals. If more than one model is specified, it will print 
+#'   as many plots as the number of models in a column of plots. Alternatively,
+#'   if \code{type = "forest"} is specified, coefficients from one or more models 
+#'   will be grouped together and displayed as a forest plot.
+#' @param theme The \code{\link{theme}} argument can be used to customomise 
+#'   the non data component of your plot. The default \code{\link{theme}} is
+#'   \code{theme_bw} and it can be substituted by any other theme compatible 
+#'   with \pkg{ggplot2}.
+#' @param signif.light Color of outer confidence intervals for significant model terms.
+#' @param signif.medium Color of inner confidence intervals for significant model terms.
+#' @param signif.dark Color of point estimates and labels for significant model terms.
+#' @param insignif.light Color of outer confidence intervals for insignificant model terms.
+#' @param insignif.medium Color of inner confidence intervals for insignificant model terms.
+#' @param insignif.dark Color of point estimates and labels for insignificant model terms.
+#' 
+#' @return Coefficient plot as a \pkg{ggplot2} \code{gg} object if \code{file = FALSE}. 
+#'   \code{NULL} otherwise.
+#' 
+#' @inheritParams texreg
+#' @inheritParams matrixreg
+#'
+#' @author Claudia Zucca, Philip Leifeld
+#' @family texreg
+#' @seealso \code{\link{texreg-package}} \code{\link{extract}}
+#'   \code{\link{extract-methods}} \code{\link{texreg}}
+#'   
+#' @examples
+#' # example from the 'lm' help file:
+#' ctl <- c(4.17,5.58,5.18,6.11,4.50,4.61,5.17,4.53,5.33,5.14)
+#' trt <- c(4.81,4.17,4.41,3.59,5.87,3.83,6.03,4.89,4.32,4.69)
+#' group <- gl(2,10,20, labels = c("Ctl","Trt"))
+#' weight <- c(ctl, trt)
+#' lm.D9 <- lm(weight ~ group)
+#' plotreg(lm.D9)    # plot model output as a diagram
+#' # plot model output as a diagram customising theme and title and automatically saving in pdf.
+#' plotreg(lm.D9, theme = theme_dark(),  ggtitle = "my title", file = "myplot.pdf")
+#' unlink("myplot.pdf")
+#' 
+#' @export
+plotreg <- function(l,
+                    file = NULL,
+                    custom.model.names = NULL,
+                    custom.title = NULL,
+                    custom.coef.names = NULL,
+                    custom.note = NULL,
+                    override.coef = 0,
+                    override.se = 0,
+                    override.pval = 0,
+                    override.ci.low = 0,
+                    override.ci.up = 0,
+                    omit.coef = NULL,
+                    reorder.coef = NULL,
+                    ci.level = 0.95,
+                    use.se = FALSE,
+                    type = "facet",
+                    theme = theme_bw(),
+                    custom.coef.map = NULL,
```

Comment:
Can you put the `custom.coef.map` argument in the right position? See the order of arguments in the `matrixreg` function.